ui/component/claimTitle/view.jsx

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react';
+import TruncatedText from 'component/common/truncated-text';
+
+type Props = {
+  uri: string,
+  claim: ?Claim,
+  title: string,
+};
+
+function ClaimTitle(props: Props) {
+  const { title, claim } = props;
+
+  return (
+    <div className="claim-preview-title">
```

Comment:
can component name match this class name

Reply:
yes, fixed up some names in https://github.com/lbryio/lbry-desktop/pull/3590